locales/fr.json

```diff
@@ -0,0 +1,20 @@
+{
+  "Execute binaries from npm packages.\n%s": "Exécutez des binaire à partir de paquets de npm .\n%s",
+  "Package to be installed.": "Paquet à installer .",
```

Comment:
I'm double checking punctuation rules in french, I just discovered that I'm writing bad french for years :) let me fix and rebase ...